git_patrol_gce.py

```diff
@@ -90,15 +95,25 @@ def main():
   # handle parse errors etc.
   with open(os.path.join(args.config_path, args.config), 'r') as f:
     raw_config = f.read()
-  git_patrol_config = yaml.load(raw_config)
+  git_patrol_config = yaml.safe_load(raw_config)
   git_patrol_targets = git_patrol_config['targets']
 
   # Connect to the persistent state database.
   loop = asyncio.get_event_loop()
-  db_pool = loop.run_until_complete(
-      asyncpg.create_pool(
-          host=args.db_host, port=args.db_port, user=args.db_user,
-          password=args.db_password, database=args.db_name))
+  db_pool = None
+  for i in range(DB_CONNECT_ATTEMPTS):
+    try:
+      db_pool = loop.run_until_complete(
+          asyncpg.create_pool(
+              host=args.db_host, port=args.db_port, user=args.db_user,
+              password=args.db_password, database=args.db_name))
+    except:
```

Comment:
Is there any way this exception can be narrowed down to a subset of possible exception types?

Reply:
Done.